src/legacy/types/contact.ts

```diff
@@ -144,6 +146,39 @@ export type FullAutocompleteResponse = {
 	};
 };
 
+export type shareFolderRoleOptions = {
```

Comment:
Use UpperCamelCase for types
```suggestion
export type ShareFolderRoleOptions = {
```